### ISSUE TITLE
[Installation] Error loading fixtures/starters.json

### ISSUE THREAD

**Issue by XaviP** [FreedomCoop/valuenetwork#78]
Error loading fixtures on install with *./manage.py loaddata ./fixtures/starters.json*
```
django.core.serializers.base.DeserializationError: Problem installing fixture '/home/user/app/valuenetwork/fixtures/starters.json': UseCase matching query does not exist.
```

**Comment by bhaugen** [comment]
@fosterlynn do you have some new and better fixtures?

**Comment by fosterlynn** [comment]
The fixture file is recent, but is from django 1.4, so probably don't work any more.  I could make a new one, or for now, can we just take the sqlite database from testocp for local development machines?  I think that would be the best solution for now given all the other priorities.

**Comment by XaviP** [comment]
Yes, this is not a prioririty, just a reminder when we have time.
